src/liboslexec/oslexec_pvt.h

```diff
@@ -835,7 +833,7 @@ class ShadingSystemImpl
 
     ustring m_colorspace;                 ///< What RGB colors mean
     ColorSystem m_colorsystem;            ///< Data for current colorspace
-    OCIOColorSystem m_ocio_system;        ///< OCIO color system (when OIIO_HAS_COLORPROCESSOR=1)
+    std::shared_ptr<OIIO::ColorConfig> m_colorconfig;  ///< OIIO/OCIO color configuration
```

Comment:
Just checking... is any CUDA_ARCH needed around this?

Reply:
I don't think so. Builds fine for me.